core/domain/user_services.py

```diff
@@ -696,10 +711,11 @@ def get_username(user_id):
     Returns:
         str. Username corresponding to the given user_id.
     """
-    if user_id == feconf.MIGRATION_BOT_USER_ID:
-        return feconf.MIGRATION_BOT_USERNAME
-    else:
-        return get_user_settings(user_id, strict=True).username
+    if (user_id == feconf.SYSTEM_COMMITTER_ID or
```

Comment:
Add backend tests for stuff you modify to make sure it works correctly not only now, but also in the future.

Reply:
Yup, thanks :) Done.